### DISCUSSION TITLE
__non_webpack_import__ && __non_webpack_import.meta

### DISCUSSION THREAD

**Discussion by Airkro** [webpack/webpack#13738] [Ideas]
like: https://webpack.js.org/api/module-variables/#__non_webpack_require__-webpack-specific

```js
__non_webpack_import__() === import()     // raw
__non_webpack_import.meta === import.meta // raw

import.meta.raw // maybe?
```

**Comment by alexander-akait** [comment]
hm, make sense I think, feel free to open an issue

**Reply by Airkro** [reply]
Before doing that, I want some help with the API design. I need more voices from the community.

**Reply by sokra** [reply]
You find `__non_webpack_require__` in `webpack/lib/APIPlugin.js` and you can add it there.

One question might be if it should always be `import` or change depending on the `output.importFunctionName` option.
